playfab-docs/features/authentication/aad-authentication/index.md

```diff
@@ -10,11 +10,11 @@ keywords: playfab, analytics, metrics, webhooks, events
 ms.localizationpriority: medium
 ---
 
-# Azure Active Directory Authentication for Playfab Game Manager
+# Microsoft Authentication for Playfab Game Manager
 
-PlayFab now supports two methods of user authentication. The first is the original PlayFab user authentication system. The second is Azure Active Directory (Azure AD).
+PlayFab now supports two methods of user authentication. The first is the original PlayFab user authentication system. The second is a Microsoft's authentication services Microsoft account and Microsoft Entra ID (formely know as Azure Active Directory).
```

Comment:
there's a redundant 'a' here

Also, maybe we could take advantage of some punctuation to highlight the difference between the ideas of Microsoft account and Entra ID

"The second is Microsoft's authentication services, 'Microsoft Account' and 'Microsoft Entra ID (formerly known as Azure Active Directory)'.

Reply:
@RonaldHarding - thanks. I have updated them